### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by FewerElk** [ScreamingSandals/BedWars#467] [Q&A] [answered]
In features there is "Other BedWars variants: CakeWars/EggWars/AnchorWars".
How can I set it ?

**Comment by Misat11** [comment, marked as answer]
You are able to create these gamemodes using BedWars, yet there is no command to just set the variant. However, you can use a different shop file for villagers, you can add extra resources in config and use them in the specific game mode and you can set the target block ("bed") to any block you want. Additionaly, Cake, Respawn Anchor and Dragon Egg have different behaviour if set as target block: 
* Cake has to be eaten in order to be destroyed.
* Respawn Anchor uses the vanilla mechanic, you need to put glowstone in order to be able to respawn. When a player respawns, the anchor level is decreased. The capacity is 4, and when it is empty, the players can be eliminated. Alternatively you can completely destroy the block, but that requires some patience.
* Dragon Egg basically does not teleport so you are able to destroy it.
